FrontDesk/tests/UnitTests/Core/AggregatesEntities/AppointmentTests/Appointment_UpdateTime.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using FrontDesk.Core.Aggregates;
+using PluralsightDdd.SharedKernel;
+using Xunit;
+
+namespace UnitTests.Core.AggregatesEntities.AppointmentTests
+{
+  public class Appointment_UpdateTime
+  {
+    [Fact]
+    public void UpdatesTimeSucceeded()
```

Comment:
`UpdatesTimeRange` is a better name. Succeeded is implied.

Need another test for what it does if the new time is the same as the old time.
Need another test for confirming an `AppointmentUpdatedEvent` is added to `Events`.